pubspec.yaml

```diff
@@ -75,6 +75,7 @@ dependencies:
   # The following adds the Cupertino Icons font to your application.
   # Use with the CupertinoIcons class for iOS style icons.
   cupertino_icons: ^1.0.0
+  pedantic: ^1.9.2
```

Comment:
I think we should use a specific version for this library `1.9.2`, not a dynamic version `^1.9.2`. `^` means that when you run `pub get` it might install a newer version than `1.9.2` if available. I think this is not good because we have CI for this, with newer `pedantic` library version it might fail CI job at anytime

Reply:
Got it. But I like that failed, we need to update each time we failed